portal/v2/src/ClusterDetailListComponents/ClusterOperatorsWrapper.tsx

```diff
@@ -97,6 +123,11 @@ export function ClusterOperatorsWrapper(props: WrapperProps) {
     <Stack>
       <Stack.Item grow>{error && errorBar()}</Stack.Item>
       <Stack>
+        <CommandBar
```

Comment:
I believe that the `<CommandBar/>` component that's used across these components here has a common-enough definition to be worth extracting into a shared wrapper component to deduplicate a lot of this change. This can be a follow-up refactor, however.